dist/rpm/os-autoinst-devel-test.spec

```diff
@@ -0,0 +1,25 @@
+%define         short_name os-autoinst-devel
+Name:           %{short_name}-test
+Version:        4.6
+Release:        0
+Summary:        Test package for %{short_name}
+License:        GPL-2.0-or-later
+BuildRequires:  %{short_name} == %{version}
+ExcludeArch:    %{ix86}
+
+%description
+.
+
+%prep
+# workaround to prevent post/install failing assuming this file for whatever
```

Comment:
assuming this file... the sentence seems incomplete? what does this mean?

Reply:
just leave it. We had that already in other places :)